ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -52,6 +62,15 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
       this.can.can('connect target', target),
     );
 
-    return { targets: targets, totalItems };
+    // Query all targets for defining filtering values
+    const options = { pushToStore: true };
```

Comment:
Shouldn't this be false?

Reply:
It for sure should be false. Thanks for catching that. Fixing now.